src/Standards/Generic/Sniffs/CodeAnalysis/UselessOverridingMethodSniff.php

```diff
@@ -93,30 +101,29 @@ public function process(File $phpcsFile, $stackPtr)
         $next = $phpcsFile->findNext(Tokens::$emptyTokens, ($next + 1), null, true);
 
         // Skip for invalid code.
-        if ($next === false || $tokens[$next]['code'] !== T_DOUBLE_COLON) {
+        if ($tokens[$next]['code'] !== T_DOUBLE_COLON) {
             return;
         }
 
-        // Find next non empty token index, should be the function name.
+        // Find next non empty token index, should be the parent method name.
         $next = $phpcsFile->findNext(Tokens::$emptyTokens, ($next + 1), null, true);
 
         // Skip for invalid code or other method.
-        if ($next === false || $tokens[$next]['content'] !== $methodName) {
+        if (strcasecmp($tokens[$next]['content'], $methodName) !== 0) {
```

Comment:
Kudos on using `strcasecmp()` here as "case-insensitive" label names in PHP are only case-insensitive for the ASCII letters and `strcasecmp()` handles this exactly right.

I suspect there are quite a few places in the code base where a `strtolower($name) === $name` type comparison is used instead of `strcasecmp()`. I've opened issue #372 with a task list to review all of those.

Reply:
Thanks for opening the new issue. I will try to help with that as well in the near future.

Reply:
Was tempted to just go and fix them, but figured it would be better to open the ticket to allow for (new) contributors to have some more starting points. ;-)